models/action_recognition/model_templates/custom-action-recognition/README.md

```diff
@@ -42,22 +42,48 @@ python ../../tools/instantiate_template.py ${MODEL_TEMPLATE} ${WORK_DIR}
 
 ### 3. Prepare data
 
-The training script assumes the data for action recognition is provided as list of videos splitted on train/val subsets.
+The training script assumes the data for the action recognition is provided as raw frames stored in directories where each directory represents a single video source.
+Additionally we assume that the data is splitted on train/val subsets.
```

Comment:
split split split

Reply:
I can't promise that it will not happen again - rules are not for me...